src/python/pants_release/git.py

```diff
@@ -36,3 +46,37 @@ def git_fetch(rev: str) -> str:
     """Fetch rev (e.g. branch or a SHA) from the upstream repository and return its SHA."""
     git("fetch", MAIN_REPO, rev)
     return git_rev_parse("FETCH_HEAD")
+
+
+def github(*args: None | str) -> str:
```

Comment:
Yes, GitHub != git, but I thought putting these helpers here for now was fine.

Reply:
Have you tried? https://github.com/PyGithub/PyGithub I haven't, but it might be marginally better than untyped subprocess code.

Reply:
I've only used it for read-only interactions. I've never tried using it to create a PR. Subprocess code is icky... but it is to GH's official CLI... so I'm in two minds.

Reply:
lolol I didn't realize that I suggested PyGitHub :sweat_smile: sorry for the double-face.

FWIW I think the new way PyGItHub+GH auth is beautiful